docs/docsite/rst/getting_started/get_started_inventory.rst

```diff
@@ -6,44 +6,40 @@ Building an inventory
 
 Inventories organize managed nodes in centralized files that provide Ansible with system information and network locations.
 Using an inventory file, Ansible can manage a large number of hosts with a single command.
-Inventories also help you use Ansible more efficiently by reducing the number of command-line options you need to specify.
-For example, inventories usually contain the SSH user so you do not need to include the ``-u`` flag when running Ansible commands.
 
-In the previous section, you added managed nodes directly to the ``/etc/ansible/hosts`` file.
-Now let's create an inventory file that you can add to source control for flexibility, reuse, and for sharing with other users.
+To complete the following steps, you will need the IP address or fully qualified domain name (FQDN) of at least one host system.
+For demonstration purposes, the host could be running locally in a container or a virtual machine.
+You must also ensure that your public SSH key is added to the ``authorized_keys`` file on each host.
 
-.. note::
-   Inventory files can be in ``INI`` or ``YAML`` format.
-   For demonstration purposes, this section uses ``YAML`` format only.
+Continue getting started with Ansible and build an inventory as follows:
 
-Complete the following steps:
-
-#. Open a terminal window on your control node.
-#. Create a new inventory file named ``inventory.yaml`` in any directory and open it for editing.
-#. Add a new group for your hosts then specify the IP address or fully qualified domain name (FQDN) of each managed node with the ``ansible_host`` field.
-   The following example adds the IP addresses of three virtual machines in KVM:
+#. Open a terminal window in your ``ansible_quickstart`` directory.
+#. Create a file named ``inventory.yaml`` and open it for editing.
```

Comment:
#. Create the ``~/ansible_quickstart/inventory.yaml `` file.

Reply:
I prefer the original change, but with the addition of adding ansible_quickstart if you plan on deleting step 1.